### PR TITLE
#167190521 Integrate HoundCI with Badge Attached to Readme

### PR DESCRIPTION
**Pull request by ayodejiAA**
#### What does this PR do?
- The PR adds hound file to the repository and attach hound badge to the readMe file. It automatically validates every changes in the codebase with ESlint style guide. 

#### Description of Task proposed in this pull request?
- [x] Add a hound file
- [x] Add hound badge link to readme file

#### What are the relevant pivotal tracker stories?
- STORY-ID: [167190521](https://www.pivotaltracker.com/story/show/167190521)

#### What I have learned working on this feature: 
- Learnt how to stash changes in the repository before rebasing.

### CODE REVIEW

**Review by chialuka — APPROVED** [review]

**Review by Adekoreday — APPROVED** [review]
great job

**Review by JuwonAbiola — APPROVED** [review]

**Review by dinobi — COMMENTED** [review]
@ayodejiAA  Why do we have changes on the README.md file

**Comment by ayodejiAA** [comment]
> @ayodejiAA Why do we have changes on the README.md file

I added houndCi badge link to the top of the file and while saving, prettier added new lines to the file.

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```
Comment (dinobi) [inline .hound.yml:3]: please add this `ignore_file: .eslintignore` so hound does look beyond eslint

**Comment by dinobi** [comment]
Going forwards don't commit a change you didn't make

File: .hound.yml (hunk 1 above)
Comment (ayodejiAA) [reply .hound.yml:3]: I will add that to the file now.

**Comment by ayodejiAA** [comment]
> Going forwards don't commit a change you didn't make

Okay, @dinobi, I will always check for changes before making commits.

**Review by dinobi — APPROVED** [review]

**Pull request merged**